src/crudeditor-lib/middleware/notifications/ExpandableNotice.react.js

```diff
@@ -18,6 +23,45 @@ export default class ExpandableNotice extends PureComponent {
     collapsed: true
   }
 
+  componentDidMount() {
+    const { id } = this.props;
+    const el = this.notificationRef;
+
+    // this is used to carry notification object from original `remove` to deferred one
+    let notification;
+
+    function handleMouseLeave() {
+      NotificationManager.remove = _removeNotification;
+    }
+
+    function handleMouseLeaveAfterTrigger() {
+      _removeNotification(notification)
+    }
+
+    function deferredRemove() {
+      if (arguments.length === 1 && (arguments[0] || {}).id === id) {
+        notification = arguments[0];
+        el.addEventListener('mouseleave', handleMouseLeaveAfterTrigger);
+      } else {
+        // in case if signature of NotificationManager.remove ever changes -> fall back to original function
+        _removeNotification(notification)
```

Comment:
output a warning to console for a developer about NotificationManager API changes